ayekan/exporters/puppetdb-exporter-deployment.yaml

```diff
@@ -0,0 +1,128 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    o11y.eu/exporter: "kafka"
```

Comment:
How is this related to kafka?

Reply:
copy/pate issues and too many open branches I think :sweat_smile: